subsys/bluetooth/mesh/gen_dtt_srv.c

```diff
@@ -195,7 +195,7 @@ struct bt_mesh_dtt_srv *bt_mesh_dtt_srv_get(const struct bt_mesh_elem *elem)
 	const struct bt_mesh_comp *comp = bt_mesh_comp_get();
 	uint16_t index;
 
-	index = elem->rt->addr - comp->elem[0].rt->addr;
+	index = elem - comp->elem;
```

Comment:
There is macro in Zephyr for this: https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/sys/util.h#L167-L185

Reply:
`comp->elem` is pointer. Macro will throw compile error.

Reply:
Resolved offline, cannot pass pointer to array in the macro